lib/entries.js

```diff
@@ -96,77 +92,78 @@ function storage(name, storage, pushover) {
 
   // store new documents using the storage mechanism
   function create (docs, fn) {
-      with_collection(function(err, collection) {
-        if (err) { fn(err); return; }
-        // potentially a batch insert
-        var firstErr = null,
-            totalCreated = 0;
-
-        docs.forEach(function(doc) {
-            collection.update(doc, doc, {upsert: true}, function (err, created) {
-                firstErr = firstErr || err;
-                totalCreated += created;
-            });
-            sendPushover(doc);
+    with_collection(function(err, collection) {
+      if (err) { fn(err); return; }
+      // potentially a batch insert
+      var firstErr = null,
+          totalCreated = 0;
+
+      docs.forEach(function(doc) {
+        collection.update(doc, doc, {upsert: true}, function (err, created) {
+          firstErr = firstErr || err;
+          if (++totalCreated >= docs.length) {
+            fn(firstErr, totalCreated, docs);
```

Comment:
I don't like this fix at all, but it seems to work. I'm hoping someone has a better idea...